README.md

```diff
@@ -1,23 +1,49 @@
 # Mux's Swift Upload SDK
-This SDK makes it easy to upload videos to Mux from an iOS app. It handles large files by breaking them into chunks and uploads each chunk individually. Each file that gets uploaded will get sent to an [upload URL created by a backend server](https://docs.mux.com/guides/video/upload-files-directly). **Do not include credentials to create an upload URL from an app.**
+This SDK makes it easy to upload videos for ingest to Mux from an iOS or iPadOS application. It handles large files by breaking them into chunks and uploading each chunk individually.
+
+Each video is uploaded to an authenticated upload URL [upload URL created by a trusted backend server request to the Mux Video API](https://docs.mux.com/guides/video/upload-files-directly). **Do not include secret API credentials to create an authenticated upload URL in an application.**
 
 ## Usage
-To use this SDK, you must first add it as a dependency. The Mux Swift Upload SDK for iOS is available on SPM.
+To use this SDK, you'll need to add it as a dependency using either Swift Package Manager or Cocoapods.
 
 ### Server-Side: Create a Direct Upload
 
-To start an upload, you must first create an [upload URL](https://docs.mux.com/guides/video/upload-files-directly). Then, pass return that direct-upload PUT URL to your app, so the app can begin the upload process
+If you haven't yet done so, you must create an [access token](https://docs.mux.com/guides/system/make-api-requests#http-basic-auth) to complete these steps.
+
+Creating a direct upload
+
```

Comment:
suggestion(non-blocking): Consider removing this since it's in the section description
```suggestion

```